### ISSUE TITLE
Fail to dpkt.pcap.Reader(f)

### ISSUE THREAD

**Issue by mattigot** [kbandla/dpkt#268]
Hey

I am trying to iterate over a iee80211 pcap file i created with wireshark and i am getting the following error:

Traceback (most recent call last):
  File "tests.py", line 6, in <module>
    pc = dpkt.pcap.Reader(f)
  File "/usr/lib/pymodules/python2.7/dpkt/pcap.py", line 105, in __init__
    self.dloff = dltoff[self.__fh.linktype]
KeyError: 127

```
   1 import dpkt
   2
   3 filename='/home/tester/pypcap/a.pcap'
   4
   5 f = open(filename)
   6 pc = dpkt.pcap.Reader(f)
   7
   8 for ts, pky in pc:
   9     print ts, len(pkt)
```

I am using Ubuntu 3.14.17 & Python 2.7.6
Can i please get some help ? :)

Thanks
Matt

**Comment by kbandla** [comment]
Matt, could you share the pcap?

**Comment by RemiDesgrange** [comment]
Start by opening the file like this : 
`f = open(filename, 'rb') # or just 'b'`

**Comment by kbandla** [comment]
what @RemiDesgrange suggested is most probably going to fix this issue. Going to close it. Lets reopen if required

**Comment by M4N0V3Y** [comment]
@kbandla, @RemiDesgrange, @mattigot 
Hello folks!

The error was  persisting for me:
Exception has occurred: AttributeError
module 'dpkt' has no attribute 'pcap'
  File "/home/m4nov3y/pcap-demo/read_cal.py", line 24, in <module>
    for ts, pkt in dpkt.pcap.Reader(open(in_file,'r')):

So, I got the following solution:
=====================
SOLVED: 

Solved when I get version 1.9.1 installed  in Python 3.6.9, like this:
`pip3 install dpkt==1.9.1`

**Comment by obormot** [comment]
> @kbandla, @RemiDesgrange, @mattigot Hello folks!
> 
>  AttributeError module 'dpkt' has no attribute 'pcap' 

this error is unrelated to the original issue
